src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -64,11 +65,18 @@ public function getItem(string $resourceClass, $id, string $operationName = null
         }
 
         $manager = $this->managerRegistry->getManagerForClass($resourceClass);
+        $classMetadata = $manager->getClassMetadata($resourceClass);
+        $metadataIdentifier = $classMetadata->getIdentifier();
         if (null === $manager) {
             throw new ResourceClassNotSupportedException();
         }
 
-        $identifierValues = explode('-', $id);
+        $identifierValues = (array) $id;
```

Comment:
We use it as an array later in the code. Do we have to cast it in a other way ?
L130
```php
 $firstIdentifier = reset($identifiers);
 foreach ($identifiers as $identifier => $value) {
```